src/schema/paddle.ts

```diff
@@ -231,6 +359,71 @@ export const resolvers: IResolvers<unknown, AuthContext> = traceResolvers<
         items,
       };
     },
+    plusPricingMetadata: async (
+      _,
+      { variant = DEFAULT_PLUS_METADATA }: PlusMetadataArgs,
+      ctx: AuthContext,
+    ): Promise<PlusPricingMetadata[]> =>
+      await getPlusPricingMetadata(ctx.con, variant),
+    plusPricingPreview: async (_, __, ctx): Promise<PlusPricingPreview[]> => {
+      const user = await ctx.con.getRepository(User).findOneOrFail({
+        where: { id: ctx.userId },
+        select: { createdAt: true },
+      });
+      const gb = getUserGrowthBookInstance(ctx.userId, {
+        subscribeToChanges: false,
+        attributes: { registrationDate: user.createdAt.toISOString() },
```

Comment:
@capJavert @idoshamun should we add `AllocationClient` here? This feature is never going to be called from the FE, so they might not get allocated if we experiment on the values.

Reply:
If that is the case I think yes, good catch

Reply:
Added! 🚀